packages/@sparrow-workspaces/src/features/rest-explorer/components/response-body/ResponseBody.svelte

```diff
@@ -32,12 +34,22 @@
       : ''}"
   >
     {#if apiState.bodyLanguage === RequestDataTypeEnum.IMAGE}
-      <!-- 
-        --
-        -- Reponse content-type set to image,
-        -- 
-      -->
-      <SparrowLogo />
+      {#if !imageHasError}
+        <!-- render image from base64 -->
+        <img
+          src={`data:${contentType};base64,${response.body}`}
+          alt="Sparrow logo"
+          on:error={() => (imageHasError = true)}
```

Comment:
can we make  a complete src string at one place to have a single abstracted data and push it to body.